presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSessionProperties.java

```diff
@@ -13,26 +13,34 @@
  */
 package io.prestosql.plugin.kafka;
 
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 import io.prestosql.spi.connector.ConnectorSession;
 import io.prestosql.spi.session.PropertyMetadata;
 
 import javax.inject.Inject;
 
 import java.util.List;
+import java.util.Map;
 
 public final class KafkaSessionProperties
 {
     private static final String TIMESTAMP_UPPER_BOUND_FORCE_PUSH_DOWN_ENABLED = "timestamp_upper_bound_force_push_down_enabled";
+    @VisibleForTesting
+    public static final String EMPTY_FIELD_STRATEGY = "empty_field_strategy";
     private final List<PropertyMetadata<?>> sessionProperties;
 
     @Inject
-    public KafkaSessionProperties(KafkaConfig kafkaConfig)
+    public KafkaSessionProperties(KafkaConfig kafkaConfig, @ForKafka Map<String, List<PropertyMetadata<?>>> extraProperties)
```

Comment:
What if we introduce something like `SessionPropertiesProvider` (like we implement in JDBC Connector) and handle like that ?